### ISSUE TITLE
File Upload: after 1st file uploaded using drag and drop, can't use drag and drop for further uploads

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/dataverse#4401]
The margins around the drag and drop window become very small, and even tho you visibly see the "+green" sign as you attempt to drag and drop a file, you get no file; resorted to using "select file to upload"

**Comment by djbrooke** [comment]
Thanks @mheppler for checking in with Sonia about this!

**Comment by mheppler** [comment]
I was able to replicate the issue of the large, grey "Drag and drop files here" msg in the upload component is not being displayed again after uploading files. (See attached screenshot.)

However, I was not able to replicate not being able to drag and drop a second file. While it is difficult to target the narrow grey-bordered upload component while trying to drag and drop without the large grey text msg, I was able to successfully drag and drop multiple files both on my localhost and dvn-build.

A fix to return the "Drag and drop files here" msg into the upload component after uploading a file has been committed.

![screen shot 2018-01-09 at 4 24 04 pm](https://user-images.githubusercontent.com/687227/34743939-8e728208-f559-11e7-9fd8-b4b34dd001e8.png)
